app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Admin::UsersController < ApplicationController
+  before_action :current_user
+  before_action :admin_user
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @users = User.all.order("created_at DESC").select(:id, :name, :email, :admin)
+    @users = @users.page(params[:page]).per(10)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to admin_users_path, notice: t('view.models.admin.new')
+    else
+      render :new
+    end
+  end
+
+  def edit
+  end
```

Comment:
中身のないメソッドは1行で記載することが可能です。
また、このメソッドでは`render :edit`というedit.html.erbをレンダリングするという処理が省略されている点にも注意してください。
アクションと別名のviewファイルをレンダリングしたい場合は、createアクションのelse句のように対象のviewファイルを明示する必要があります。
```suggestion
  def edit; end
```